src/blueprint.zig

```diff
@@ -372,6 +374,228 @@ pub const Pattern = struct {
 	}
 };
 
+fn generatePropertyEnum() type {
```

Comment:
Functions returning a type should start with a capital letter.
Also why even make this a function when you alias the result to `Property` anyways?
```zig
const Property = blk: {
	...
	break :blk @Type(...);
};
```